### PR TITLE
govuk classes for anchor, list and paragraph elements

### PR DESCRIPTION
**Pull request by agalamatis**
trello: https://trello.com/c/RzRAhmef/33-3-use-govuk-link-govuk-list-and-govuk-body-styles-in-brief-responses-frontend

### CODE REVIEW

File: app/templates/briefs/application_submitted.html [hunk 1]
```diff
@@ -36,14 +36,14 @@ <h1 class="govuk-heading-l">What happens next<h1>
...
-        <p>The buyer will tell you if you're not successful.</p>
+        <p class="govuk-body">When the opportunity closes, the buyer will score your evidence. If you’re one of the top {{ brief.get('numberOfSuppliers') }} suppliers, you'll go through to the evaluation stage.</p>
+        <p class="govuk-body">The buyer will tell you if you're not successful.</p>
 
         <h2 class="heading-xmedium">Evaluation</h2>
 
         <div class="explanation-list">
-          <p class="lead">At the evaluation stage, the buyer will ask you to provide:</p>
-          <ul class="list-bullet">
+          <p class="lead govuk-body">At the evaluation stage, the buyer will ask you to provide:</p>
```
Comment (lfdebrux) [inline app/templates/briefs/application_submitted.html]: Instances of class `lead` should be replaced with `govuk-body-l`

```suggestion
          <p class="lead govuk-body">At the evaluation stage, the buyer will ask you to provide:</p>
```

Comment (agalamatis) [reply app/templates/briefs/application_submitted.html]: thanks, I made the changes in the last commit

Comment (lfdebrux) [reply app/templates/briefs/application_submitted.html]: If you could squash that last commit into the previous one changing paragraphs that would be great

File: app/templates/frameworks/opportunities_dashboard.html [hunk 2]
```diff
@@ -31,8 +31,8 @@ <h1 class="govuk-heading-l">Your {{ framework.name }} opportunities</h1>
         </div>
     </div>
 
-  <p>
-    <a href="{{ url_for('external.list_opportunities', framework_family=framework.family) }}">
+  <p class="govuk-body"></p>
```
Comment (aliuk2012) [inline app/templates/frameworks/opportunities_dashboard.html]: Do we need this empty `p` tag?

File: app/templates/briefs/check_your_answers.html [hunk 3]
```diff
@@ -60,13 +60,13 @@ <h1 class="govuk-heading-l">
   </div>
 </div>
 
-<p class="padding-bottom-small">&nbsp;</p>
+<p class="padding-bottom-small govuk-body">&nbsp;</p>
```
Comment (aliuk2012) [inline app/templates/briefs/check_your_answers.html]: do we need this? looks like an existing hack just to add spacing. Happy to pair on this issue.

File: app/templates/briefs/application_submitted.html [hunk 4]
```diff
@@ -57,25 +57,25 @@ <h2 class="heading-xmedium">Evaluation</h2>
...
+        <p class="govuk-body">
           The buyer will score all suppliers who reached the evaluation stage using the weightings they published with their requirements. They’ll provide feedback if you’re unsuccessful.
         </p>
       </div>
 
   </div>
 </div>
 
-<p class="padding-bottom-small">&nbsp;</p>
+<p class="padding-bottom-small govuk-body">&nbsp;</p>
```
Comment (aliuk2012) [inline app/templates/briefs/application_submitted.html]: Another existing spacing hack we should try remove

**Pull request merged**